app/main/appium-method-handler.js

```diff
@@ -109,7 +109,7 @@ export default class AppiumMethodHandler {
     return {variableName, variableType, strategy, selector, elements};
   }
 
-  async _execute ({elementId, methodName, args, skipScreenshotAndSource}) {
+  async _execute ({elementId, methodName, args, skipContextsScreenshotAndSource}) {
```

Comment:
`skipContextsScreenshotAndSource` is now getting really long, probably should refactor into a name that is short but descriptive, maybe `skipMetadataRetrieval` or just `skipRefresh`?

Reply:
updated to `skipRefresh `